### DISCUSSION TITLE
continous profiling of memray

### DISCUSSION THREAD

**Discussion by zdyj3170101136** [bloomberg/memray#530] [Q&A]
if i want to continous profiling my python process.

i should resuce the memray overhead as much as i can.

is there have any solution?

**Comment by zdyj3170101136** [comment]
there is two solution:
1, run memray attach run pid periodicity.

2,memray export a http hanlder, i call the api, and then it return the memory profiling data.

**Comment by godlygeek** [comment]
Memray is really designed for offline profiling, not continuous profiling. You can find some previous discussion about the difficulties in supporting continuous profiling with Memray on this thread: https://github.com/bloomberg/memray/issues/445#issuecomment-1707254170
